Dockerfile

```diff
@@ -1,2 +1,17 @@
-FROM node:20.12.2-alpine3.19 AS base
-RUN wget -qO- https://get.pnpm.io/install.sh | ENV="$HOME/.shrc" SHELL="$(which sh)" sh -
+FROM node:20.12.2-alpine3.19
+
+USER 0:0
+
+RUN npm install -g pnpm@v9.8.0 \
```

Comment:
Can't we use corepack to install pnpm, based on the version specified in package.json?